admin-console/src/main/java/com/agiletec/apsadmin/portal/PageAction.java

```diff
@@ -678,7 +682,7 @@ protected String checkDelete(String selectedNode) throws ApsSystemException {
 		} else if (!isUserAllowed(currentPage) || !isUserAllowed(currentPage.getParent())) {
 			this.addActionError(this.getText("error.page.remove.notAllowed"));
 			return "pageTree";
-		} else if (currentPage.getChildren().length != 0) {
+		} else if (currentPage.getChildrenCodes().length != 0) {

```

Comment:
There is a null != check on this earlier in the class (line 645). Is one needed here?

Reply:
I'll fix it